### PR TITLE
amber 1.3.2 (new formula)

### PR DESCRIPTION
**Pull request by crimson-knight**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by carlocab** [comment]
```
  /home/linuxbrew/.linuxbrew/opt/binutils/bin/ld: cannot find -lsqlite3 (this usually means you need to install the development package for libsqlite3): No such file or directory
```

File: Formula/amber.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+class Amber < Formula
+  desc "Crystal web framework. Bare metal performance, productivity and happiness"
+  homepage "https://amberframework.org/"
+  url "https://github.com/amberframework/amber/archive/refs/tags/v1.3.2.tar.gz"
+  sha256 "37511d6b4afe308e1943cedeab9114b01d5787d868c23d2c0cc555917a21c830"
+  license "MIT"
+
+  depends_on "crystal"
```
Comment (carlocab) [inline Formula/amber.rb:8]: This should fix your Linux failure:
```suggestion
  depends_on "crystal"
  uses_from_macos "sqlite"
```
When you apply this suggestion, please squash the change into your existing commit and use the commit message
```
amber 1.3.2 (new formula)
```
Thanks!

**Review by carlocab — DISMISSED** [review]
LGTM, thanks! I'll leave this for a bit to give other maintainers a chance to review.

**Review by chenrui333 — DISMISSED** [review]
LGTM too

**Comment by carlocab** [comment]
Looks like we have some permissions misconfiguration here. Let me sort that out internally first.

**Pull request merged**